doc/controller/controller.rst

```diff
@@ -1,3 +1,31 @@
-Controller class
+Controller
 =====================================
-To be filled.
+The controller is responsible for providing set points for the assets in the simulator.
+The controller parses the ESDL and creates objects of the assets it is providing set points for.
```

Comment:
The controller parses ESDL?    The ESDL is parsed by ```simulator-core```  to create an object model of the ESDL which is used by our solver.  This parser now also creates the controller objects

Reply:
indeed, since the controller also needs information from the esdl